src/ScratchStorage.js

```diff
@@ -171,6 +183,27 @@ class ScratchStorage {
             tryNextHelper();
         });
     }
+
+    /**
+     * Store an asset by type & ID.
+     * @param {AssetType} assetType - The type of asset to fetch. This also determines which asset store to use.
+     * @param {DataFormat} [dataFormat] - Optional: load this format instead of the AssetType's default.
+     * @param {DataFormat} [data] - Optional: store this data, rather than the data in the asset
+     * @param {?string} assetId - The ID of the asset to fetch: a project ID, MD5, etc.
+     * @return {Promise.<id, title>} A promise for asset metadata
```

Comment:
Is this type annotation correct? I thought `.<x,y>` was used to denote instances of parameterized type (e.g. for maps it would be the type of the keys and the values respectively)...

Reply:
I found this example on the JSDoc web site: `@return {Promise<string>}`

I think the proper way to do this might be something like: `@return {Promise<{id: string, title: string}>}` (or is the `id` a `Number`?)

Reply:
The stuff in the returned object could be anything the web store returns so I think `Promise.<object>` is what we want.